kernel/include/rendering.h

```diff
@@ -0,0 +1,43 @@
+#pragma once
+#include <stdint.h>
+
+typedef enum {
+    e_RGB8,
+    e_BGR8,
+    e_BitMask,
+    e_BltOnly,
+    e_FBFormatMax,
+} FBFormat;
+
+typedef struct Framebuffer {
+    void* address;
+    uint64_t width;
+    uint64_t height;
+    uint64_t pixels_per_scanline;
+    FBFormat format;
+} __attribute__((packed)) Framebuffer;
+
+extern uint64_t g_bg_color;
+extern uint64_t g_fg_color;
+extern Framebuffer* g_frame_buffer;
+
+void put_pixel(uint64_t x, uint64_t y, uint32_t color);
+void put_char(char c, uint64_t x, uint64_t y, uint32_t fg, uint32_t bg);
+
+//print value in hex, where size is determined and returned
```

Comment:
Make this comments a bit more descriptive. Think of it a bit like documentation.  

What is size referring to in the comment?
Do the x and y arguments specify pixel coordinates or something else?